spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -719,6 +714,12 @@ def run(self, run_time):
                 self._do_load()
                 loading_done = True
 
+        # Disable auto pause and resume if the binary can't do it
+        if (self._executable_start_type !=
+                ExecutableStartType.USES_SIMULATION_INTERFACE):
+            self._config.set("Buffers", "use_auto_pause_and_resume",
+                             "False")
```

Comment:
OK, so then this is not the correct fix ;).  The issues are:

1. The check on line 666 will be invalid.  This will allow it to use auto pause and resume when the binary isn't a simulation binary, which won't work.
1. Executable start type won't be set at all when using a virtual board.  This seems dangerous (although it looks like it will get the right answer)